routers/web/repo/release.go

```diff
@@ -83,7 +83,18 @@ func releasesOrTags(ctx *context.Context, isTagList bool) {
 		ctx.Data["PageIsTagList"] = false
 	}
 
-	tags, err := ctx.Repo.GitRepo.GetTags()
+	listOptions := models.ListOptions{
+		Page:     ctx.FormInt("page"),
+		PageSize: ctx.FormInt("limit"),
+	}
+	if listOptions.PageSize == 0 {
+		listOptions.PageSize = setting.Repository.Release.DefaultPagingNum
```

Comment:
I think we should have a separate pagination number from release. Release will include many notes and downloads, but tags will only include source downloads.
I would like to set 1 for releases but 10 for tags on https://gitea.com .

Reply:
this is not the scope of this pull :)

I can create another one ...